src/widgets/requirements/ui/requirements.test.tsx

```diff
@@ -35,6 +37,6 @@ describe('Requirements', () => {
     const button = screen.getByRole('link', { name: /Register as a mentor/i });
```

Comment:
I can't write comments for lines above - my suggestions:
- add test-case for checking the component is visible using `data-testid`
- use your text constant from your `constants.ts` file instead strings (or you can use mocked-data in `.test.tsx` file)
- use it.each instead many `expect`

Reply:
If I use constants in the tests where I have taken the text from the component, the tests will always succeed - the source of the text for the code and for the test is the same.
Should I put the text in the tests into a separate constant(s) for use in the tests?

Reply:
We usually use mock-data for tests, but we can also use constants from a data-file, the original source of the data is there - if we change the content, we don't want to rewrite the tests too. Moreover, I'm not sure that we need to test text content at all, we need to discuss this on Mon. In order not to block the SSG, let's leave the strings as is for now. Please, fix just 1st & 3rd points

Reply:
Done

Reply:
Please, use `it.each` from [Vitest](https://vitest.dev/api/#test-each)
You can use construction like
`it.each([[string1], [string2]])(
    'should render component with string %s',
    s => {
      expect(screen.getByText(s)).toBeVisible();
    },
  );`

Reply:
Done